app/wp-content/themes/kstg/library/scss/layout/_vehicle.scss

```diff
@@ -1,16 +1,24 @@
 .vehicle {
-  padding: 3.3%;
+  padding: 0 5% 80px;
   background-color: $light-grey;
-  h2 {
-    font-size: 1.6em;
-    margin: 25px 0;
+  clear: both;
+  .container {
```

Comment:
Are you trying to get this so that it works on all pages with the `.container` class? If so, should we move this to another file and not nest it under `.vehicle`?